File: 3581-TheTwoSneakyNumbersOfDigitville/3581-TheTwoSneakyNumbersOfDigitville.js
// Last updated: 3/6/2026, 2:08:15 PM
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
    let newMap = new Map();
    let res = [];

    for(let i=0;i<nums.length;i++){
        if(newMap.has(nums[i])){
            res.push(nums[i]);
        }
        else{
            newMap.set(nums[i]);
        }
    }
    return res;
};

File: 3581-TheTwoSneakyNumbersOfDigitville/3581-TheTwoSneakyNumbersOfDigitville.cpp
// Last updated: 7/17/2026, 4:23:45 PM
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        unordered_map<int , int> newMap;
        vector<int> res;

        for(int i=0;i<nums.size();i++){
            if(newMap.find(nums[i]) != newMap.end()){
                res.push_back(nums[i]);
            }
            else{
                newMap[nums[i]]=1;
            }
        }
        return res;
    }
};